__tests__/integration/txLogger.js

```diff
@@ -0,0 +1,146 @@
+/* eslint-disable no-console */
+
+import * as Path from 'path';
+
+const fsold = require('fs');
+
+const fs = fsold.promises;
+
+export const loggers = {
+  /**
+   * @type: TxLogger
+   */
+  test: null
+};
+
+/**
+ * A logger for every transaction on the integration tests for debugging.
+ */
+export class TxLogger {
```

Comment:
Why are we implementing a custom logger? Why don't we use `winston` that is already a dependency on the project?

Reply:
The rational on implementing a _Transaction Logger_ was to encapsulate common messages such as `"Tx {id} moving funds from {a} to {b}"` in a easily reusable way.

The method chosen on the low level was the `fs.appendFile` for its simplicity on the Proof of Concept phase, but I agree a change to using the `winston` would be more consistent with the rest of the application.

Reply:
Adapted the `TxLogger` to use `winston` underneath.
The overall quality of the logs has improved as a consequence, with more metadata being easily readable on the logs :+1: